src/Test/intlinear.jl

```diff
@@ -681,6 +681,244 @@ function indicator4_test(model::MOI.ModelLike, config::TestConfig)
     end
 end
 
+function semiconttest(model::MOI.ModelLike, config::TestConfig{T}) where T
```

Comment:
These tests seem to differ in only a few places, could you refactor with an auxiliary function
```julia
function _semitest(model::MOI.ModelLike, config::TestConfig{T}, int::Bool) where T
    [...]
end
semiconttest(model::MOI.ModelLike, config::TestConfig) = _semitest(model, config, false)
semiinttest(model::MOI.ModelLike, config::TestConfig) = _semitest(model, config, true)
```